Arithmetic
  import java.util.Scanner;

public class Arthimetic_Exception {
    public static void main(String[] args)
    {
        Scanner in=new Scanner(System.in);
        try
        {
            System.out.println("Enter two numbers :");
            int a =in.nextInt();
            int b =in.nextInt();
            int  c=a/b;
            System.out.print(c);
        }
        catch (Exception e)
        {
            System.out.print("There is exception :\n" +e);
        }
    }
}

Array
public class ArrayOutOfBound {
    public static void main(String[] args) {
        int a[] = {1, 2, 3, 4, 5};
        try {
            for (int i = 0; i <= 6; i++) {
                System.out.println(a[i]);
            }

        }
        catch(Exception e)
        {
            System.out.println("Exception :"+e.getMessage());
        }
    }
}
nullpoint
public class Exception {
    public static void main(String[] args) {
        try {
            String str = null;

            int length = str.length();
        } catch (NullPointerException e) {
            System.out.println("NullPointerException caught!");
            e.printStackTrace();
        }
    }
}
File not found
import java.io.File;
import java.util.Scanner;

public class FileNotFound {
    public static void main(String[] args) {
        try {
            File myFile = new File("test.txt");
            Scanner myReader = new Scanner(myFile);
        } catch (Exception e) {
            System.out.println("An error occurred."+e);

        }
    }
}
